modules/auth_email/module.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
there is now a default config field on modules. we should use that where it makes sense.